specifyweb/settings/__init__.py

```diff
@@ -141,6 +141,11 @@
     ('config', SPECIFY_CONFIG_DIR),
 )
 
+# Add web app manifest
+WEBPACK_LOADER = {
+    'MANIFEST_FILE': os.path.join(SPECIFY_CONFIG_DIR, "/static/manifest.json"),
```

Comment:
What does this do? Do we need to add a python dependency for it to work? (https://github.com/django-webpack/django-webpack-loader)

Is this required? You might be able to achieve the same by defining a view that would serve the manifest.json page:

View: https://github.com/specify/specify7/blob/3dced0964721218de64659e8371b1c218c6d3fef/specifyweb/urls.py#L28

it's imported here: https://github.com/specify/specify7/blob/3dced0964721218de64659e8371b1c218c6d3fef/specifyweb/urls.py#L11

the view itself is defined like this:
(https://specifydev.slack.com/archives/D02LXH9PEUU/p1672952503649319)
https://github.com/specify/specify7/blob/3dced0964721218de64659e8371b1c218c6d3fef/specifyweb/frontend/views.py#L20-L29

Reply:
Did you plan to make any change related to this? I don't see the commit

This is what I get right now:
<img width="1800" alt="Screenshot 2023-01-10 at 22 28 01" src="https://user-images.githubusercontent.com/40512816/211717771-e4a21209-2a8e-4f9c-a214-da50ab0694c5.png">

Reply:
Never mind, all is good. It's just that the static files directory in docker does not update automatically when you switch branches, so I had to manually copy the newly added files into it.

Reply:
Though, Chrome fails to pick up the icon for me:

<img width="372" alt="image" src="https://user-images.githubusercontent.com/40512816/211959768-0b8bae4e-8efc-4d9e-b222-856f30a5c307.png">

(even in incognito)

Looks like you told it in specify.html to look at `/static/img/favicon.ico`
Instead, it should be `/favicon.ico`.

I have just pushed that change, and also some template refactoring so that favicons are present on all pages, not just on the main page (the login page, and the choose collection)

Reply:
I was under the impression anytime a URL ended in `favicon.ico` it would redirect to the one specified in the `urls.py` but I see that is not the case. Thank you!

Reply:
`url(r'^favicon.ico', RedirectView.as_view(url='/static/img/short_logo.svg'))`
only redirects if the location is exactly `/favicon.ico` then

Reply:
ah good point. that's a subtle difference.

If you remove `^`, then it would match any URL that contains `favicon.ico`

To many any URL that ends with it, you have to say `r'favicon.ico$'`